distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector/Extensions.enso

```diff
@@ -0,0 +1,91 @@
+import project.Any.Any
+import project.Data.Vector.No_Wrap
+import project.Data.Vector.Vector
+import project.Internal.Array_Like_Helpers
+
+## PRIVATE
+   ICON dataframe_map_column
+   Applies a function to each element of the vector, returning the `Vector` of
+   results. Unlike `Vector.map`, `map_no_wrap` does not wrap errors in
+   `Map_Error`.
+
+   Arguments:
+   - function: A function that takes an element in the vector and returns
+       some transformation of that element.
+
+   > Example
+       Add 1 to each element of the vector.
+
+           [1, 2, 3] . map +1
+Vector.map_no_wrap : (Any -> Any) -> Vector Any
```

Comment:
Every project that needs this kind of `map_no_wrap` extension shall define it for itself. `map_no_wrap` shalln't be visible to Enso end users.